app/assets/stylesheets/application.css.scss

```diff
@@ -116,3 +116,61 @@ i.fa.medium {
 ::-webkit-input-placeholder {
   color: $placeholder-text;
 }
+
+.pagecontent {
+  align-items: flex-start;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  margin: 0 auto;
+  max-width: 980px;
+  padding: 0 20px;
+  position: relative;
+}
+
+.resourcearticle {
+  flex: 1;
+  margin-top: 0;
+  max-width: 740px;
+}
+
+h1 {
+  margin-top: 0;
+}
+
```

Comment:
@aaron-suarez I definitely think that this rule targeting all `h1` elements across the app should be switched out for something more specfic.  I don't think it's a good practice to globally apply this CSS rule everywhere, it will make things harder to maintain moving forward and lead to unintended side-effects, or require other CSS rules to eventually be added to overwrite this rule.

Maybe add a class to the `h1` you want to target for this pull request, and give it the `margin-top: 0` rule, or use some other type of nesting to specify.